### PR TITLE
Add oc_createcronjob&createjob functionality

### PR DESCRIPTION
**Pull request by kasturinarra**
Add lib for oc create cronjob & oc create job

### CODE REVIEW

**Comment by kasturinarra** [comment]
@zhouying7780 please help review, thanks !!

File: lib/rules/cli/4.1.yaml [hunk 1]
```diff
@@ -1372,3 +1372,26 @@
...
+  :cmd: oc create job <name>
+  :options:
+    :allow_missing_template_keys: --allow-missing-template-keys=<value>
+    :dry_run: --dry-run=<value>
+    :from: --from=<value>
+    :image: --image=<value>
+    :output: --output <value>
+    :template: --template=<value>
+    :save_config: --save-config=<value>
+    :validate: --validate=<value>
```
Comment (liangxia) [inline lib/rules/cli/4.1.yaml:1397]: Please sort the command and options in alphanumeric order

File: lib/rules/cli/4.1.yaml [hunk 2]
```diff
@@ -1372,3 +1372,26 @@
   :options:
     :skip-check: --skip-check=<value>
     :z: -z default
+:create_cronjob:
```
Comment (zhouying7780) [inline lib/rules/cli/4.1.yaml:1375]: Please place the command in alphabetical order.

Comment (kasturinarra) [reply lib/rules/cli/4.1.yaml]: Done

File: lib/rules/cli/4.1.yaml (hunk 1 above)
Comment (kasturinarra) [reply lib/rules/cli/4.1.yaml]: Done

**Comment by zhouying7780** [comment]
@liangxia  Please help review again and merge , thanks .

**Pull request merged**